test/k8sT/istio.go

```diff
@@ -101,6 +101,14 @@ var _ = Describe("K8sIstioTest", func() {
 
 		ProvisionInfraPods(kubectl)
 
+		By("Disabling debug mode for Istio test")
+		err := kubectl.CiliumExecAll("cilium config Debug=Disabled")
```

Comment:
Why? It will be hard to debug in case something fails.

Reply:
Debug mode puts more load on the system. While I was running the Istio tests (which revealed the bug that this PR is fixing), I was encountering severely high load (values up to 42). I can drop these Istio CI commits and work exclusively on fixing the Istio tests in another branch if you like.

Reply:
Saved the commits in a separate branch and dropped them from this PR.